decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
That was a nice catch! So we just learned that this RuboCop auto-correction is not 100% safe. I'm happy I didn't break all your installations :stuck_out_tongue:...

So should I change this to `to_str`? I wonder if it's better to leave it as it is and disable the cop inline explaining the issue with a comment, so that nobody in the future will attemp this kind of change again.